vagrant/provisioners/install-lucee.sh

```diff
@@ -63,7 +63,7 @@ sudo /bin/cp -f /vagrant/configs/lucee-server.xml /opt/lucee/lib/lucee-server/co
 sudo cp /vagrant/configs/server.xml /opt/lucee/tomcat/conf/server.xml
 
 sudo mkdir -p /opt/lucee/config/web
-sudo cp -f /vagrant/configs/web.xml /opt/lucee/tomcat/conf/web.xml
```

Comment:
The file `web.xml` is actually correct.  It is needed to override the web context location to the folder path created on the above line.  The issue is that the file apparently didn't get committed to our repo.  If you put this line back, I'll merge the rest of the pull.  I'll grab a correct version of the web.xml, or you can try your hand at it :)